include/net/arp.h

```diff
@@ -17,6 +17,12 @@
 
 #include <net/ethernet.h>
 
+/**
+ * @brief Address resolution (ARP) library
+ * @defgroup arp ARP Library
```

Comment:
Currently the functions are not documented as the they are not meant to be called by user applications. This might change in the future thou and in that case the defgroup is there already.